portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/usecase/WYSIWYGUsecase.testcase

```diff
@@ -191,6 +191,13 @@ definition {
 			groupName = "WYSIWYG Site Name",
 			title = "Web Content Title");
 
+		JSONDocument.addFileWithUploadedFile(
+			dmDocumentDescription = "DM Document Description",
+			dmDocumentTitle = "Document_2.jpg",
```

Comment:
Not sure if this is a typo, Document_1.jpg? It doesn't affect the test but just looks inconsistent.

Reply:
Looks trivial enough to forward now and update this later if needed.

Reply:
Yes, that's a typo, I will fix it later.